userland/examples/lua-hello/Makefile

```diff
@@ -0,0 +1,20 @@
+# Makefile for user application
+
+# Specify this directory relative to the current application.
+TOCK_USERLAND_BASE_DIR = ../..
+
+# Which files to compile.
+C_SRCS := $(wildcard *.c)
+
+# External libraries used
+EXTERN_LIBS += $(TOCK_USERLAND_BASE_DIR)/lua53
```

Comment:
Hrm... OK. Except the Lua C-api doesn't tend to be compatible across major versions... So it would almost certainly break existing apps anyway